src/openforms/forms/tests/variables/test_variables_in_logic_trigger.py

```diff
@@ -119,6 +120,49 @@ def test_static_variable_in_trigger(self, m_get_static_data):
 
         self.assertEqual(status.HTTP_201_CREATED, response.status_code)
 
+    @patch(
+        "openforms.forms.models.FormVariable.get_static_data",
+        return_value=[
+            FormVariable(
+                name="Authentication identifier",
+                key="auth_identifier",
+                data_type=FormVariableDataTypes.object,
+                initial_value={
+                    "plugin": "digid",
+                    "attribute": AuthAttribute.bsn,
+                    "value": "123456782",
+                },
+            )
+        ],
+    )
```

Comment:
can we replace this mocking structure with replacing the variable registry instance? that decouples it from the actual implementation and makes our tests a bit more robust